lib/main.coffee

```diff
@@ -51,32 +54,32 @@ module.exports =
     settingsView?.dispose()
     settingsView?.remove()
     settingsView = null
+    packageManager = null
 
   consumeStatusBar: (statusBar) ->
-    PackageManager = require './package-manager'
-    packageManager = new PackageManager()
-    Promise.all([packageManager.getOutdated(), packageManager.getInstalled()]).then (values) ->
-      outdatedPackages = values[0]
+    packageManager ?= new PackageManager() # TODO: Why is this needed when the package manager is initialized at the top of the file?
```

Comment:
All the relevant specs (except the first one) fail because packageManager is null without this line.